platform/viewer/src/lib/DICOMSR/isToolSupported.js

```diff
@@ -0,0 +1,8 @@
+import * as dcmjs from 'dcmjs';
```

Comment:
Does dcmjs allow us to scoped import `adapters`?

Reply:
Does it automatically include all adapters, even if we're not using a library it may have an adapter for?